README.adoc

```diff
@@ -68,17 +62,20 @@ mvn install liberty:start-server
 
 Maven builds the application and runs it inside Open Liberty.
 
-Point your browser to the `http://localhost:9080/inventory/systems` URL.
-Because you just started the application, the inventory is currently empty. Point to the
+Point your browser to the `http://localhost:9080/inventory/systems` URL to access the `inventory`
+service. Because you just started the application, the inventory is currently empty. Then, access the
 `http://localhost:9080/inventory/systems/localhost` URL to add the localhost host into the inventory.
 
-Next, point to the `https://localhost:9443/metrics` MicroProfile Metrics URL. Click the `Advanced`
-button to proceed. Use the `confAdmin` and `microprofile` credentials to log in. You can see both
-the standard metrics and the application metrics in a text format.
+Access the `inventory` service i.e. the `http://localhost:9080/inventory/systems` URL at least once
+for application metrics to be collected. Otherwise, the metrics will not appear.
+
+Next, point your browser to the `https://localhost:9443/metrics` MicroProfile Metrics endpoint. Login
+as the `confAdmin` user with `microprofile` as the password. You can see both the system and application
+metrics in a text format.
 
 To see only the application metrics, point your browser to `https://localhost:9443/metrics/application`.
-Because these metrics are application-related, you have to first to point to the `http://localhost:9080/inventory/systems`
-URL, otherwise they will not appear. See sample outputs for the `@Timed`, `@Gauge`, and `@Counted` annotations:
+
+Sample outputs for the `@Timed`, `@Gauge`, and `@Counted` metrics:
```

Comment:
A colon is essentially like a period. We need a complete sentence before it. How about, "**See the following** sample outputs for the `@Timed`, `@Gauge`, and `@Counted` metrics: